alpine/Dockerfile

```diff
@@ -16,7 +16,10 @@ RUN adduser -Su 1337 kong \
 	&& rm -rf /tmp/usr \
 	&& cp -R /tmp/etc / \
 	&& rm -rf /tmp/etc \
-	&& apk del .build-deps
+	&& apk del .build-deps \
+	&& chown -R kong /usr/local/kong \
+	&& chgrp -R 0 /usr/local/kong \
+	&& chmod -R g=u /usr/local/kong
```

Comment:
I think the dockerfile should gain a USER directive

Reply:
I don't think it should. Default behaviour is to run as root then su-exec to Kong. Open shift behaviour is to run as a random UID and continue as that UID